### DISCUSSION TITLE
Making movies with many plot states from the command line

### DISCUSSION THREAD

**Discussion by nDimensionalSpace** [LLNL/Silo#295] [Q&A] [answered]
I generally make my animations from the command line, as

visit -movie -format png -sessionfile <sessionname> -output blah,

and some other options.  I am noting that the files that are output are of the format "blah%4d.png", which may be problematic for databases with more than 10,000 plot states.  Is there a way to modify the output file name format from the command like?  I searched the help docs and couldn't find any such thing . . .  Thanks.

**Comment by brugger1** [comment, marked as answer]
The number of digits is hard wired at 4 digits. I'll take a look at enhancing the script to just use the correct number of digits based on the number of files needed to create the movie (maybe with a minimum of 4, so as not to change the existing behavior, except when necessary).

**Reply by markcmiller86** [reply]
@brugger1 apologies for not seeing this. It looks like you've answered. But, this was in Silo's repo and I think appropriate place is VisIt's repo, right? If so, unfortunately, there is no easy way to migrate this discussion. GitHub supports migrations only *within* an organization and not across them. @nDimensionalSpace just a reminder to take care to post issues with VisIt and its supporting tools to [VisIt's discussions](https://github.com/visit-dav/visit/discussions). This is for issues with Silo 😉.

**Reply by nDimensionalSpace** [reply]
😬

Sorry about that . . .  I will be more cognizant in the future.
